### PR TITLE
Sm engine

### PR DESCRIPTION
**Pull request by intsco**
isotope_image_correlation - first image mask returned
measure_of_chaos small update (approved by Andy)

### CODE REVIEW

**Comment by intsco** [comment]
Hey guys, can I expect merging this changes into the master at some point?

**Comment by intsco** [comment]
Should we add a new version or what is your policy?

**Comment by andy-d-palmer** [comment]
Good idea.

File: pyImagingMSpec/image_measures.py [hunk 1]
```diff
@@ -33,12 +33,9 @@ def measure_of_chaos(im, nlevels, overwrite=True, statistic=None):
     """
     statistic = statistic or _default_measure
     # don't process empty images
-    if np.sum(im) == 0:
+    if np.sum(im) <= 0:
```
Comment (DarthPumpkin) [inline pyImagingMSpec/image_measures.py]: Why?

Comment (intsco) [inline pyImagingMSpec/image_measures.py]: Just in case)

Comment (DarthPumpkin) [inline pyImagingMSpec/image_measures.py]: But so far we have treated negative entries as zeros, right?

**Pull request merged**